torch/tensor.py

```diff
@@ -388,7 +388,7 @@ def __ipow__(self, other):
         raise NotImplementedError("in-place pow not implemented")
 
     def __rpow__(self, other):
-        return self.new([other]) ** self
+        return self.new_tensor(other) ** self
```

Comment:
shouldn't this be `as_tensor`? Also, `new_tensor` throws a warning when the input is already a tensor...

Reply:
If `other` is a tensor, then this would be dispatch to the __pow__ function instead right?

Reply:
Oh you are correct! Sorry!

Reply:
No worries :smile: